src/components/QuoteBubble/DesktopShare.js

```diff
@@ -0,0 +1,39 @@
+import React from 'react';
+
+import { socialMediaVariant } from '../../types';
+import { Facebook, Twitter } from '../Icons';
+import Hitbox from '../Hitbox';
+
+const socialMediaData = {
```

Comment:
I think it's a bit too complicated considering what it does. I think using it as `<DesktopShare url={...}><Twitter /></DesktopShare>` in ButtonsGroup would be cleaner.
If you don't want to change it maybe use dynamic keys notation, so they could be extracted from consts: `[TWITTER]: {...}`

Reply:
I would stick with the object inside `DesktopShare`, to avoid logic specified for Desktop Sharing on the higher component level. According to that change, we also need to change `MobileShare` too, for keeping consistency. With this approach `BubbleButtons` can act as a simple container.